src/patternfly/components/FormControl/form-control.scss

```
@@ -179,7 +178,10 @@ $pf-c-form-control--m-search--Coordinates: "M505 442.7L405.3 343c-4.5-4.5-10.6-7
 
   &.pf-m-search {
     padding-left: var(--pf-c-form-control--m-search--PaddingLeft);
```

Making the change above to `--pf-c-form-control--m-search--BackgroundPosition` would allow you to use `--pf-c-form-control--PaddingLeft` here

```suggestion
    --pf-c-form-control--PaddingLeft: var(--pf-c-form-control--m-search--PaddingLeft);
```